### ISSUE TITLE
RVC Export Onnx Issue

### ISSUE THREAD

**Issue by benc1221** [w-okada/voice-changer#241]
im having an issue exporting my pth to onnx its like everytime i click export onnx i get greeted with please wait (about 1 min) and it has been over 30 minutes with no directory save or anything.

**Comment by w-okada** [comment]
Can you chare the console output? Any error message?

If message shows out of memory, please use cpu export. Set gpu "-1" and exportOnnx.

**Comment by benc1221** [comment]
![Screenshot 2023-05-22 053139](https://github.com/w-okada/voice-changer/assets/89867041/7d6cb683-feaa-4787-88bb-242aad1e6070)

**Comment by w-okada** [comment]
It seems export onnx successfully.
How about dialog which says "please wait"?

**Comment by benc1221** [comment]
Please wait (about 1 min) and it never exports

**Comment by benc1221** [comment]
[18552:0522/102003.171:ERROR:gpu_init.cc(523)] Passthrough is not supported, GL is disabled, ANGLE is

what is causing this error because it is keeping me from exporting

**Comment by grimoire-vc** [comment]
> [18552:0522/102003.171:ERROR:gpu_init.cc(523)] Passthrough is not supported, GL is disabled, ANGLE is
> 
> what is causing this error because it is keeping me from exporting

I don't think this is what is causing you to not be able to export. That error happens for me every time after a couple minutes of having voice-changer open. It's unrelated.

I've also experienced being unable to export to onnx on my laptop most of the time, though it works fine on my desktop. When I try to export to onnx on my laptop, all 8GB of VRAM is used up until I close voice-changer. It doesn't appear that my GPU or CPU are being used during that time either. I have been able to export to onnx once on my laptop, but I'm not sure why it worked that time. It was after accidentally trying to export an onnx model to onnx, and then switching to the PyTorch model I actually intended to export to onnx. That time it worked normally.
